packages/buckram/assets/styles/components/specials/_footnotes.scss

```diff
@@ -26,7 +26,7 @@
   .footnote {
     display: prince-footnote;
     padding-left: $footnote-padding-left;
-    margin-left: 0;
+    margin-left: $footnote-blockquote-margin-left;
```

Comment:
@ricardopressbooks This proposed fix does move the footnote block out of the page margin, but we shouldn't use the $footnote-blockquote-margin-left variable, as it's meant to be used for "blockquote elements with a class of fn-blockquote"  https://buckram.pressbooks.org/#specials-variable-footnote-blockquote-margin-left. If we want to use a variable, we should probably create a new one (maybe "$footnote-margin-left") in https://github.com/pressbooks/pressbooks-book/blob/0994860a20a1e69f4e16dac286d4e3332f9b2ce4/packages/buckram/assets/styles/variables/_specials.scss#L79-L96

Reply:
Thank you @SteelWagstaff definitely add a new variable is the a better option. I added the fix in this branch.